sbi/utils/restriction_estimator.py

```diff
@@ -168,6 +168,8 @@ def __init__(
         self._prior = prior
         self._classifier = None
 
+        self._device = "cpu"  # TODO hot fix to prevent the tests from crashing
```

Comment:
Shall I remove this, too?

Reply:
You can leave it in for now, I'll have to check what's going wrong later

Reply:
not sure, but it could be that the `RestrictionEstimator` just needs the `_device` `kwarg`, or, if it is actually never using `self._device` (e.g., no GPU training implemented), just set `training_device="cpu"` in line 207

Reply:
We made the same observation. It does not seem to be a difficult bug.